internal/generator/services.go

```diff
@@ -89,6 +89,9 @@ func composeToServiceValues(
 	if composeServiceValues.Labels != nil {
 		lagoonType = lagoon.CheckServiceLagoonLabel(composeServiceValues.Labels, "lagoon.type")
 	}
+	if lagoonType == "" {
+		return ServiceValues{}, fmt.Errorf("No Lagoon type has been set for service %s. If a Lagoon type is not required for this service, please set the Lagoon type to 'none'", composeService)
+	}
 	if lagoonType != "" {
```

Comment:
Probably change this to an else rather than doing the second if to check if it is not "".
```
} else {
```

Reply:
Updated now